### PR TITLE
Scalable Snowman Workshop (Workshop Bounty Resubmission #1)

### PR DESCRIPTION
**Pull request by JakeGerber**
I added left and right movement for the snowmen. Turtle does not seem to have any key down function (which is really annoying) so you have to keep pressing the keys.

### CODE REVIEW

**Comment by JakeGerber** [comment]
Previous: #1466

File: workshops/scalable_snowman_workshop/README.md [hunk 1]
```diff
@@ -0,0 +1,353 @@
...
+  
+  for x in range(3):
+    turtle.forward(radius2/2)
+    turtle.begin_fill() 
+    turtle.circle(radius1/6) 
+    turtle.end_fill()
+  
+  turtle.left(90)
+```
+- We are setting our direction to down (turtle.setheading(270)) and going to the top of the first body part.
```
Comment (faisalsayed10) [inline workshops/scalable_snowman_workshop/README.md:179]:
```suggestion
- We are setting our direction to down (`turtle.setheading(270)`) and going to the top of the first body part.
```

Comment (faisalsayed10) [reply workshops/scalable_snowman_workshop/README.md:179]: it shud be highlighted as a code.

**Review by faisalsayed10 — CHANGES_REQUESTED** [review]
Hey!
I just reviewed your workshop, we are very close to merge it but it needs some very few improvements.
- There is a lack of explanation overall. Please try to add more explanation to each and every code blocks.

- I recommend you to use the markdown syntax for images as you have used ``

Yeah, that's it!

**Pull request closed without merging**